test/unit/utils.js

```diff
@@ -1,8 +1,25 @@
 import chai from 'chai';
-import { normalizeQuerySort } from '../../src/utils';
+import { generateUUIDv4, normalizeQuerySort } from '../../src/utils';
+import sinon from 'sinon';
 
 const expect = chai.expect;
 
+describe('generateUUIDv4', () => {
+	beforeEach(() => {
+		sinon.restore();
+	});
+
+	it('generates a UUID manually when crypto is unavailable', () => {
+		sinon.spy(Math, 'pow');
+		sinon.spy(Math, 'random');
+		const uuid = generateUUIDv4();
+		expect(uuid).to.be.a('string');
+		expect(uuid.length).to.equal(36);
+		expect(Math.pow.calledWithMatch(2, 8)).to.be.true;
+		expect(Math.random.callCount).to.be.equal(16);
+	});
```

Comment:
Globally mocking crypto was a tough chore, due to chai/mocha's logic. (With Jest it would've been a different story, way easier :|)
I couldn't figure it out at the moment. Therefore we test the fallback scenario only